testsuite/features/secondary/proxy_as_pod_basic_tests.feature

```diff
@@ -121,25 +121,105 @@ Feature: Register and test a Containerized Proxy
     And the uptime for "sle_minion" should be correct
     And I should see several text fields
 
+  Scenario: Pre-requisite: subscribe system to Fake Channel
+    Given I am on the Systems overview page of this "sle_minion"
+    When I follow "Software" in the content area
+    And I follow "Software Channels" in the content area
+    And I wait until I do not see "Loading..." text
+    And I check radio button "Fake-Base-Channel"
+    And I wait until I do not see "Loading..." text
+    And I check "Fake-Child-Channel"
+    And I click on "Next"
+    Then I should see a "Confirm Software Channel Change" text
+    When I click on "Confirm"
+    Then I should see a "Changing the channels has been scheduled." text
+    And I wait until event "Subscribe channels scheduled by admin" is completed
```

Comment:
Instead of subscribing to another base channel. Would make sense to keep the same base channel, and just be sure we have marked the `Fake-RPM-SUSE-Channel` so we can test using the patches and packages of that child channel?
If that can work, we don't need to perform a clean-up scenario where we need to subscribe again to the right base channel and make sure all the child channels are ok.

Reply:
It's a good idea but will required a more complex refactoring. It will probably requires to create the activation key differently. It's not a bad idea to have a channel change scenario. It covers more product features.

Reply:
Ok, indeed we will cover more that way, it could be harder to maintain for the next person jumping here, but it covers the subscription channel feature.

A separate improvement we can work on in the future is a scenario that passes an operating system, it can wait for the check of all the sub-products in a programmatic way, which will make it easier to maintain the scenarios where we explicitly mention the channel names in the scenario.